tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -411,21 +411,20 @@ def test_environment_paths(self):
         """
         try:
             org = make_org(cached=True)
-            payload = {
+            test_env = make_lifecycle_environment({
                 'organization-id': org['id'],
-            }
-            test_env = make_lifecycle_environment(payload)
+            })
         except CLIFactoryError as err:
             self.fail(err)
 
         # Add paths to lifecycle environments
-        result = LifecycleEnvironment.paths({'organization-id': org['id'],
-                                            'permission-type': 'readable'})
-        self.assertEqual(result.return_code, 0,
-                         "return code must be 0, instead got {0}".
-                         format(result.return_code))
-        self.assertEqual(
-            len(result.stderr), 0,
-            "There should not be an error here.")
-        self.assertIn(u'Library >> {0}'.format(test_env['name']),
-                      result.stdout)
+        result = LifecycleEnvironment.paths({
+            'organization-id': org['id'],
+            'permission-type': 'readable',
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
```

Comment:
@Ichimonji10 the output for CLI test already have that information

```
======================================================================
FAIL: @Test: Create User for all variations of Email Address
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/shiningpanda/jobs/9514c559/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-cli/tests/foreman/cli/test_user.py", line 210, in test_positive_create_user_4
    self.fail(err)
AssertionError: Failed to create User with {u'firstname': u'K2ULxiZySY', u'lastname': u'xT5VBXqEjs', u'location-ids': None, u'password': u'OaY7mNwnSS', u'auth-source-id': 1, u'default-location-id': None, u'organization-ids': None, u'default-organization-id': None, u'admin': None, u'mail': u'\uafde\U0002aff6\U000290a2\u11f1\U00029374\U0002780f\U00024a84\U00028a18\u42d9\u6d03@example.com', u'login': u'z1jD73'} data due to:
  /usr/lib/ruby/gems/1.8/gems/hammer_cli-0.1.4/lib/hammer_cli/./apipie/../abstract.rb:68: warning: already initialized constant DEFAULT_LABEL_INDENT
  [ERROR 2015-02-17 21:32:02 Exception] Email address is invalid
  Could not create the user:
    Email address is invalid
  
-------------------- >> begin captured logging << --------------------
robottelo: DEBUG: Running test User/test_positive_create_user_4_1__uafde_U0002aff6_U000290a2_u11f1_U00029374_U0002780f_U00024a84_U00028a18_u42d9_u6d03_example_com
robottelo.cli.factory: DEBUG: User "z1jD73" password not provided OaY7mNwnSS was generated
robottelo.common.ssh: DEBUG: >>> [qe-sat6-rhel66.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv user create --firstname="K2ULxiZySY" --lastname="xT5VBXqEjs" --password="OaY7mNwnSS" --auth-source-id="1" --mail="꿞𪿶𩂢ᇱ𩍴𧠏𤪄𨨘䋙洃@example.com" --login="z1jD73"
robottelo.common.ssh: INFO: Instantiated Paramiko client 0x464bb90
robottelo.common.ssh: INFO: Destroying Paramiko client 0x464bb90
robottelo.common.ssh: INFO: Destroyed Paramiko client 0x464bb90
robottelo.common.ssh: DEBUG: <<< /usr/lib/ruby/gems/1.8/gems/hammer_cli-0.1.4/lib/hammer_cli/./apipie/../abstract.rb:68: warning: already initialized constant DEFAULT_LABEL_INDENT
[ERROR 2015-02-17 21:32:02 Exception] Email address is invalid
Could not create the user:
  Email address is invalid

robottelo.cli.factory: DEBUG: /usr/lib/ruby/gems/1.8/gems/hammer_cli-0.1.4/lib/hammer_cli/./apipie/../abstract.rb:68: warning: already initialized constant DEFAULT_LABEL_INDENT
[ERROR 2015-02-17 21:32:02 Exception] Email address is invalid
Could not create the user:
  Email address is invalid

--------------------- >> end captured logging << ---------------------
```

As you can see the stderr is available on the test result as the last logging message. By adding that we will be repeating ourselves and I think in this case does not worth the extra work.